src/views/ModelPlan/TaskList/Beneficiaries/Frequency/index.tsx

```diff
@@ -0,0 +1,367 @@
+                                        }
+                                      </FieldErrorMsg>
+                                      <Field
+                                        as={TextInput}
+                                        className="maxw-none"
+                                        id="beneficiaries-beneficiarySelectionFrequency-other"
+                                        maxLength={50}
+                                        name="beneficiarySelectionFrequencyOther"
+                                      />
+                                    </div>
+                                  )}
+                              </Fragment>
+                            ))}
+                        </Fieldset>
+                        <AddNote
+                          id="beneficiaries-beneficiarySelectionFrequency-note"
+                          field="beneficiarySelectionFrequencyNote"
+                        />
+                      </FieldGroup>
+
+                      <FieldGroup
+                        scrollElement="beneficiaryOverlap"
+                        error={!!flatErrors.beneficiaryOverlap}
+                      >
+                        <Label htmlFor="beneficiaries-overlap">
+                          {t('levelOfConfidence')}
+                        </Label>
+                        <FieldErrorMsg>
+                          {flatErrors.beneficiaryOverlap}
+                        </FieldErrorMsg>
+                        <Fieldset>
+                          {Object.keys(OverlapType)
+                            .sort(sortOtherEnum)
+                            .map(key => (
+                              <Fragment key={key}>
+                                <Field
+                                  as={Radio}
+                                  id={`beneficiaries-overlap-${key}`}
+                                  name="beneficiaries-overlap"
+                                  label={translateOverlapType(key)}
+                                  value={key}
+                                  checked={values.beneficiaryOverlap === key}
+                                  onChange={() => {
+                                    setFieldValue('beneficiaryOverlap', key);
+                                  }}
+                                />
+                              </Fragment>
+                            ))}
+                        </Fieldset>
+                        <AddNote
+                          id="beneficiaries-overlap-note"
+                          field="beneficiaryOverlapNote"
+                        />
+                      </FieldGroup>
+
+                      <FieldGroup
+                        scrollElement="precedenceRules"
+                        error={!!flatErrors.precedenceRules}
+                      >
+                        <Label
+                          htmlFor="beneficiaries-precedenceRules"
```

Comment:
The convention we should try to adhere to with id's is all kebab-case.  `precedenceRules` is camelCased and should be `precedent-rules`